src/gpio.rs

```diff
@@ -469,7 +469,7 @@ macro_rules! gpio {
                     }
 
                     #[allow(dead_code)]
-                    pub(crate) fn set_alt_mode(&self, mode: AltMode) {
+                    pub fn set_alt_mode(&self, mode: AltMode) {
```

Comment:
is there any reason not to make this public? I looked at other hal crates, they made this a private function, but referenced all the pin modes as function

Reply:
I don't see a reason, but I'm not very familiar with that code. I also don't know why the `#[allow(dead_code)]` is here and on `AltMode` itself. Seems all very weird.